plugins/preflight.yaml

```diff
@@ -33,7 +33,31 @@ spec:
     sha256: "5d96a7b9abb3699fc495b8739519e7b608c1d42f8b22b1e6581b889de7376872"
     files:
     - from: "/preflight.exe"
-      to: "kubectl-preflight.exe"
-    bin: "kubectl-preflight.exe"
-  shortDescription: Executes preflight tests and application conformance verifications on a cluster before installing an application
+      to: "."
+    bin: "preflight.exe"
+  shortDescription: Executes application preflight tests in a cluster
   homepage: https://github.com/replicaetdhq/troubleshoot
+  caveats: |
+    Usage:
+      $ kubectl preflight <URL>
+
+    Flags:
+      -h, --help                          help for preflight
+          --interactive                   interactive preflights (default true)
+          --kubecontext string            the kubecontext to use when connecting (default "~/.kube/config")
+          --serviceaccount string         name of the service account to use. if not provided, one will be created
+          --format string                 output format, one of human, json, yaml. only used when interactive is set to false (default "human")
+
+    Documentation:
+      https://help.replicated.com/docs/troubleshoot/kubernetes/preflight/overview/
+
+  description: |
+    This plugin executes application-specific preflight checks and conformance tests against a cluster, prior to installation of an application.
```

Comment:
Nice.

Minor nit: can you wrap lines at ~80 characters? Currently, this will produce badly formatted output on smaller screens.